src/main/java/org/elasticsearch/bootstrap/Security.java

```diff
@@ -60,10 +52,13 @@ static void configure(Environment environment) throws Exception {
     }
 
     /** returns dynamic Permissions to configured paths */
-    static Permissions createPermissions(Environment environment) throws IOException {
+    static Permissions createPermissions(Environment environment, boolean addTempDir) throws IOException {
```

Comment:
can the 2 tests calling this with false just override java.io.tmpdir for the test? then no need for a new boolean?